content/spin/v2/cli-reference.md

```diff
@@ -3389,8 +4018,37 @@ Options:
 
 {{ blockEnd }}
 
+**Note:** For additional information, please see the [Managing Templates](/spin/managing-templates) and/or [Creating Templates](/spin/template-authoring) sections of the documentation.
```

Comment:
Something is wrong here, but I cannot point to what it is...

<img width="914" alt="image" src="https://github.com/fermyon/developer/assets/12242001/9da3092e-e2ff-4cd3-b174-d13a97a33803">

Reply:
Thanks for spotting this, let me take a look.

Reply:
Looks like there was an extra `{{ tabs "spin-version" }}` in there.
I have removed it and will test.

Reply:
![image](https://github.com/fermyon/developer/assets/9831342/e13e14e1-ba1c-42e3-8adc-972a35dfe703)